SeattleColleges/belindas-closet-nextjs
# Belindas-closet-nextjs-2-214-Signup-Page

Resolves #214 
Changed sign-up page to send a POST request to the Nest.js backend. 
To test:
local nest.js server with MongoDB need to be working, if port is different from 3000, it needs to be changed in /belindas-closet-nextjs/app/auth/sign-up/page.tsx to the one your server is using. 
Go to signup page http://localhost:8080/auth/sign-up (again, check your ports), enter the required data and click the Sign Up button, if it is successful, you'll see the alert and the response from the API with the token like this:

![Screenshot_20240215_112020a](https://github.com/SeattleColleges/belindas-closet-nextjs/assets/17448320/bbffc08d-29ec-4d05-94df-d0da4379f856)

Also there should be the new entry for the new user you signed up in the Nest database:

![Screenshot_20240215_112250](https://github.com/SeattleColleges/belindas-closet-nextjs/assets/17448320/8a3a9303-6e7d-481f-bccb-509b1bc88784)
